prebuild_binaries/build_scripts/build.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```

Comment:
Naming: instead of prebuild_binaries/build_scripts/build.sh, can this just be build/build/sh? Having "prebuild_binaries" as a top-level directory in a source tree is somewhat unusual, "build" is a more common name.

Reply:
"build" would be clobbered by the directory where the compiled binaries are placed (like, when developing the agent), so holding off on renaming.

Maybe "tools" (gRPC uses this), or "compile?

Reply:
"tools" seems appropriate.

Reply:
Done.